### ISSUE TITLE
Add id attr to the svg node in extract mode

### ISSUE THREAD

**Issue by sqal** [kisenka/svg-sprite-loader#227]
**Do you want to request a feature, report a bug or ask a question?**
Tiny feature request

**What is the current behavior?**
When using extract mode with plainSprite enabled ([official example](https://github.com/kisenka/svg-sprite-loader/blob/master/examples/interop-with-html-webpack-plugin/build/index.html)) loader generates svg without `id` attribute, unlike runtime mode where it adds  `__SVG_SPRITE_NODE__` id. Id attribute is helpful in hiding svg on the page, without it I currently need to use small workaround in form:

`<div hidden><svg>...</svg</div>`

**What is the expected behavior?**
Generate svg node with id attribute (it would be nice to have id name configurable?). Or perhaps it would be better to add a new option e.g. svgAttrs:

```
loader: 'svg-sprite-loader',
options: {
  svgAttrs: {
    id: 'sprite',
    hidden: true,
  }
}
```

**Comment by kisenka** [comment]
Agreed with you, but I want to implement total user setup over generating output in next 4.x version. This is not critical, so I add it to roadmap.

**Comment by kisenka** [comment]
@sqal BTW it was implemented in 3.6.0, see https://github.com/kisenka/svg-sprite-loader#sprite-attributes :)
